client/src/components/UI/GoogleLogin.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import queryString from 'query-string';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import GoogleButton from './GoogleButton';
+
+const styles = theme => ({
+  button: {
+    margin: theme.spacing.unit,
+  },
+  input: {
+    display: 'none',
+  },
+});
+
+class GoogleLogin extends Component {
+  state = { isLogIn: false, username: '' };
```

Comment:
Store this login in context ao that you can use islogin state in any component

Reply:
This state is primarily for button, login and logout check is achieved by checking the local storage token.

Reply:
In this updated PR I am controlling through local storage token for login button as well